app/frontend/src/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,61 @@
+import DateFnsUtils from "@date-io/dayjs";
+import {
+  KeyboardTimePicker,
+  MuiPickersUtilsProvider,
+} from "@material-ui/pickers";
+import { CHANGE_TIME } from "features/constants";
+import { Control, Controller } from "react-hook-form";
+import dayjs, { Dayjs } from "utils/dayjs";
+
+interface TimepickerProps {
+  control: Control;
+  defaultValue?: Dayjs;
+  id: string;
+  label: string;
+  name: string;
+  className?: string;
+  error?: boolean;
+  errorText?: string;
+}
+
+export default function Timepicker({
```

Comment:
I'm a bit lost as to why this doesn't work that well at all on keyboard. Tried changing the value back to using JavaScript `Date` instead and that's improved keyboard usability slightly, but for some reason you still can't switch to "PM" without opening up the dialog :/

Reply:
Also the clock seems to only allow drag and drop rather than clicking on the numbers, which makes this pretty difficult/probably less worthwhile to unit test. The author of RTL seems to suggest that too (and recommends doing this in Cypress e2e tests, which we currently don't have yet) https://github.com/testing-library/user-event/issues/440#issuecomment-685010755

Reply:
It would make things a lot simpler technically, but I feel like since we are using the Material date picker, having that side by side makes the UI looks quite inconsistent. Might it be worth using the combined time and date picker instead? Not sure if it suffers the same keyboard usability problem I'm having here with the time picker though.